docker-compose.yml

```diff
@@ -5,47 +5,47 @@ services:
     volumes:
       - .:/var/www/html/
     ports:
-      - "80:80"
```

Comment:
@m-triassi can you tell me if this will affect the file, looks like prettier went a little too ham